english/dream/dream.md

```diff
@@ -1,6 +1,7 @@
 You understand that it's difficult to wake up from a dream if you think this is all really happening to you.
 You attempt to point out things that are around you that cannot possibly occur in real life
-such the marshmallow walls and hand.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.